tests/Extension/AssetTest.php

```diff
@@ -1,13 +1,17 @@
 <?php
 
-namespace League\Plates\Extension;
+declare(strict_types=1);
```

Comment:
I think this should be removed. as it should be either applied everywhere in the project or simply nowhere.

and it cannot be applied everywhere as the PHP requirement from the project still includes 5.x